src/include/block_chunkcache.h

```diff
@@ -0,0 +1,75 @@
+/*-
+ * Copyright (c) 2014-2020 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *  All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * WiredTiger's chunk cache. Locally caches chunks of remote objects.
+ */
+
+#define WT_CHUNKCACHE_DEFAULT_HASHSIZE 32 * 1024
+#define WT_CHUNKCACHE_DEFAULT_CHUNKSIZE 1024 * 1024
+#define WT_CHUNKCACHE_DRAM 0
```

Comment:
The reason why I put DRAM, because if we ever decide to have it in NVRAM, which is also known as "memory", we would need to use the "file system" type, because managing NVRAM requires to have a file system on top. I could do WT_CHUNKCACHE_IN_VOLATILE_MEMORY?